### ISSUE TITLE
EasyFarm stops after encountering an error

### ISSUE THREAD

**Issue by GBEY0402** [EasyFarm/EasyFarm#501]
EasyFarm stops after encountering an unknown error, no message about what made it crashes...

OK, fine, an error. RESTART IT!

**Comment by massochist** [comment]
Mine crashes after the first mob and the interface itself flickers boxes and some text disappears all together, my guess is that this program has not been updated in a long time and just doesn't work anymore.

**Comment by GBEY0402** [comment]
Since it's sourcefree and i'm a developper in C# (the program writen in C#), I managed to fix my issue myself by adding some validations in the code. Your problem seems to come from 1+ bad configurations. Make sure to click auto-fill on every action. If it is marked on "Unknown" the system can't resolve the action and crash.

**Comment by massochist** [comment]
Ya that doesn't work either, if I hit autofill it just says it cant find what I'm typing which was Torcleaver, no way that isn't in the file it's not a new move. anyways I removed all abilities and just had it run straight mele still crashed.

**Comment by H0110WS0U1** [comment]
Yeah, mine says the skills can't be found in the resources. Any chance you'd be willing to upload your fixed version, GBEY0402?
